### PR TITLE
Issue/2382 extend carousel 

### PR DESCRIPTION
**Pull request by denyo**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://github.com/porscheofficial/oss-docs/blob/67c5b24838a293ce7a964884e6005eb71f2b8579/CONTRIBUTOR_LICENSE_AGREEMENT.md)

#### References

- Preview: https://designsystem.porsche.com/issue/2382/components/carousel/examples#slides-with-flexible-widths
- https://designsystem.porsche.com/issue/2382/components/carousel/examples#jump-to-slide-active-slide-index

#### Scope

Short description of the pull request scope

### CODE REVIEW

File: packages/components/src/components/carousel/carousel.tsx [hunk 1]
```diff
@@ -156,12 +171,17 @@ export class Carousel {
       // TODO: this uses matchMedia internally, since we also use it, there is some redundancy
       breakpoints: getSplideBreakpoints(this.slidesPerPage as Exclude<BreakpointCustomizable<number>, string>),
       // https://splidejs.com/guides/i18n/#default-texts
-      i18n: parseJSONAttribute(this.intl),
+      i18n: parseJSONAttribute(this.intl || {}), // can only applied initially atm
```
Comment (mhaegeleMHP) [inline packages/components/src/components/carousel/carousel.tsx:174]:
```suggestion
      i18n: parseJSONAttribute(this.intl || {}), // can only be applied initially atm
```

**Comment by affilnost** [comment]
@denyo Did you test the case of `auto-width="true"`  and "Jump to slide" at the same time? I tried it in Stackblitz, and there it didn't work.. I took the example of "Jump to slide" and added there `auto-width="true"`, and then "Jump to slide" stopped working

Bildschirm­foto 2023-03-22 um 09 32 38

**Comment by denyo** [comment]
> @denyo Did you test the case of `auto-width="true"` and "Jump to slide" at the same time? I tried it in Stackblitz, and there it didn't work.. I took the example of "Jump to slide" and added there `auto-width="true"`, and then "Jump to slide" stopped working
> 
> Bildschirm­foto 2023-03-22 um 09 32 38

I guess the slides in your example are not wide enough and therefore in the viewport without sliding? 
https://stackblitz.com/edit/f2wgfe?file=index.html

File: packages/components/src/components/carousel/carousel.tsx [hunk 2]
```diff
@@ -103,11 +105,17 @@ export class Carousel {
...
+  @Prop() public intl?: CarouselInternationalization;
 
   /** Adapts the color when used on dark background. */
   @Prop() public theme?: Theme = 'light';
 
+  /** Defines which slide to be active (zero-based numbering). */
+  @Prop() public activeSlideIndex?: number = 0;
+
+  /** If set to true, the carousel respects the width of each slide which has to be defined via CSS. This option overrides the `slidesPerPage` prop. */
+  @Prop() public autoWidth?: boolean = false;
```
Comment (stephanschroeter) [inline packages/components/src/components/carousel/carousel.tsx:117]: `slidesPerPage: number | BreakpointCustomizable<SlidesPerPage> | 'auto'`?

**Pull request merged**